spark/dl/src/test/scala/com/intel/analytics/bigdl/optim/EstimatorSpec.scala

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intel.analytics.bigdl.optim
+
+import com.intel.analytics.bigdl._
+import com.intel.analytics.bigdl.nn.{ClassNLLCriterion, Linear, Sequential}
+import com.intel.analytics.bigdl.utils.{Engine, T}
+import org.scalatest.{BeforeAndAfter, FlatSpec, Matchers}
+import com.intel.analytics.bigdl.tensor.TensorNumericMath.TensorNumeric.NumericFloat
+import org.apache.spark.SparkContext
+import org.apache.spark.ml.{DLEstimator, DLEstimatorData, MlTransformer}
+import org.apache.spark.rdd.RDD
+import org.apache.spark.sql.{DataFrame, SQLContext}
+
+class EstimatorSpec extends FlatSpec with Matchers with BeforeAndAfter {
+  val model = new Sequential[Float]()
+  var sc : SparkContext = _
+  before {
+    Engine.setNodeAndCore(1, 1)
+    Engine.init
+    val conf = Engine.createSparkConf().setAppName("Test Optimizer Wrapper").setMaster("local")
+    sc = new SparkContext(conf)
+  }
+  "A wrapper" should "executor optimizer" in {
```

Comment:
Pipeline Model (PipelineStage) persistence is not supported in 1.5 , it is supported since 1.6*